stories/Modal.stories.js

```diff
@@ -98,7 +98,6 @@ const Template = (args) => ({
 
 const base = {
     modalId: 'c05sg3G4oA4',
-    closeBtnText: 'Close',
```

Comment:
@markup-mitchell This text will still be required - all icon-only buttons need visually-hidden text for screenreaders:

```
 <span class="visually-hidden">
           {{ closeBtnText }}
  </span>
```

Reply:
Replaced the prop and updated Button to add `visually-hidden`

Reply:
@mjpvs can you look at this when you have a minute and see if I've addressed Gemma's comments?

Reply:
By changing the way this works i.e adding the visually-hidden class to the span around the slot - it means all the instances will need updated/tested where this is used as currently the visually-hidden class is added in the slot itself so will now appear twice.

Reply:
The only component using it is numberInput. I've removed the internal spans from there since it doesn't make sense to impement that behaviour independently of a button that takes iconOnly. This fixes the iconOnly button stories too.

Reply:
done